### PR TITLE
docs(yard): adds yard to select component page

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: documentation-site/components/yard/config/select.ts [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+      type: PropTypes.Boolean,
+      description: 'Defines if multiple options can be selected.',
+    },
+    noResultsMsg: {
+      value: undefined,
+      type: PropTypes.ReactNode,
+      description:
+        'Message to be displayed if no options is found for a search query.',
+    },
+    onBlur: {
```
Comment (gergelyke) [inline documentation-site/components/yard/config/select.ts]: can you reuse the changehandlers from here? https://github.com/uber-web/baseui/blob/master/documentation-site/components/yard/config/common.ts

an example can be found here: https://github.com/uber-web/baseui/blob/master/documentation-site/components/yard/config/checkbox.ts#L114

File: documentation-site/components/yard/config/select.ts [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+    'menuFontSelected',
+  ],
+  props: {
+    'aria-label': {
+      value: undefined,
+      type: PropTypes.String,
+      description: 'Sets the aria-label attribute.',
+      placeholder: 'string',
+    },
+    'aria-describedby': {
```
Comment (gergelyke) [inline documentation-site/components/yard/config/select.ts]: can we add `hidden:true`  to these properties to simplify the UI a bit?

File: documentation-site/pages/components/select.mdx [hunk 3]
```diff
@@ -40,6 +43,8 @@ Select menus in Base hover atop of a selection menu while providing a simple lis
 Proper spacing, color, and a check-mark are clear indicators of a selection. Each menu is
 scrollable, if applicable.
 
+<Yard componentName="Select" placeholderHeight={48} {...selectYardConfig} />
```
Comment (tajo) [inline documentation-site/pages/components/select.mdx]: Should be the first thing on the page.

File: documentation-site/pages/components/select.mdx [hunk 4]
```diff
@@ -30,6 +30,9 @@ import {StatefulSelect, TYPE} from 'baseui/select';
 import * as SelectExports from 'baseui/select';
 import {Block} from 'baseui/block';
 
+import Yard from '../../components/yard/index';
```
Comment (tajo) [inline documentation-site/pages/components/select.mdx]: Please remove unused `Overrides` and `API` imports

File: documentation-site/components/yard/config/select.ts [hunk 5]
```diff
@@ -0,0 +1,387 @@
+import pick from 'just-pick';
+import {Select, SIZE, TYPE} from 'baseui/select';
+import {PropTypes} from '../const';
+import {changeHandlers} from './common';
+import {TConfig} from '../types';
+
+const selectConfig: TConfig = {
```
Comment (tajo) [inline documentation-site/components/yard/config/select.ts]:
```suggestion
const SelectConfig: TConfig = {
```

Just to match all other places.

File: documentation-site/components/yard/config/select.ts [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+            "Prop passed to an option indicating it's currently selected.",
+        },
+        $size: 'size',
+        $type: 'type',
+      },
+    },
+  },
+};
+
+export default selectConfig;
```
Comment (tajo) [inline documentation-site/components/yard/config/select.ts]:
```suggestion
export default SelectConfig;
```

**Pull request merged**